src/data/query-suggestions.json

```diff
@@ -87,7 +90,7 @@
             }
           }
         },
-        "summary": "Send requests to the Algolia Query Suggestions REST API",
+        "summary": "Send requests to the Algolia REST API",
```

Comment:
Summary and description are now too generic to be useful, the source should be updated before we can accept automated updates

Reply:
Those are really escape hatch for API client users I guess they would be better of filtered on the mcp indeed